DarkScript3/GUI.cs

```diff
@@ -173,7 +174,7 @@ private void OpenToolStripMenuItem_Click(object sender, EventArgs e)
             }
             InstructionDocs oldDocs = Docs;
             OpenFileDialog ofd = new OpenFileDialog();
-            ofd.Filter = "EMEVD Files|*.emevd; *.emevd.dcx;";
+            ofd.Filter = "EMEVD Files|*.emevd; *.emevd.dcx|All files|*.*";
```

Comment:
If we're going to explicitly allow opening JS files without a corresponding EMEVD, we should probably include `*.emevd.js` and `*.emevd.dcx.js` as a filter option.

Reply:
Done